lib/user_spam_scorer.rb

```diff
@@ -0,0 +1,102 @@
+# -*- encoding : utf-8 -*-
+class UserSpamScorer
+  DEFAULT_SCORE_MAPPINGS = {
+    :name_is_all_lowercase? => 1,
+    :name_is_one_word? => 1,
+    :name_includes_non_alpha_characters? => 3,
+    :name_is_garbled? => 5,
+    :email_from_spam_domain? => 5,
+    :email_from_spam_tld? => 3,
+    :about_me_includes_currency_symbol? => 2,
+    :about_me_is_link_only? => 3,
+    :about_me_is_spam_format? => 1,
+    :about_me_includes_anchor_tag? => 1,
+    :about_me_already_exists? => 4
+  }
+
+  DEFAULT_CURRENCY_SYMBOLS = %w(£ $ € ¥ ¢)
+  DEFAULT_SPAM_DOMAINS = %w(mail.ru temp-mail.de tempmail.de shitmail.de)
+  DEFAULT_SPAM_FORMATS = [
+    /\A.*+\n{2,}https?:\/\/[^\s]+\z/,
```

Comment:
just looks like a typo of either `.*` (optionally any character)  or `.+` (at least one character)